src/topology/instances/real.lean

```diff
@@ -165,11 +165,11 @@ metric.uniform_continuous_iff.2 $ λ ε ε0, begin
 end
 
 lemma real.uniform_continuous_mul (s : set (ℝ × ℝ))
-  {r₁ r₂ : ℝ} (r₁0 : 0 < r₁) (r₂0 : 0 < r₂)
+  {r₁ r₂ : ℝ} (r₁0 : 0 < r₁)
```

Comment:
Again, the assumption `r₁0` can also be removed if you use the improved version of  `rat_mul_continuous_lemma` that I gave above.